.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1,10 @@
+contact_links:
+  - name: Questions, Suggestion or Feature requests?
+    url: https://github.com/coder/coder/discussions/new/choose
```

Comment:
I would've named it "Questions, suggestions or feature requests".

Reply:
Updated.